plugins/services/src/js/pages/task-details/ServiceTaskDetailPage.js

```diff
@@ -16,11 +17,11 @@ class ServiceTaskDetailPage extends React.Component {
     const {id, taskID} = params;
 
     let routePrefix = `/services/overview/${encodeURIComponent(id)}/tasks/${encodeURIComponent(taskID)}`;
-    const tabs = [
+    const tabs = Hooks.applyFilter('serviceTaskDetailPageTabs', [
```

Comment:
@nLight using `Mount` instead of hooks sounds great, but I'm not sure about your example. Shouldn't we use the same approach as we used for the config views?